src/app/components/forms/post/post.vue

```diff
@@ -532,11 +527,11 @@
 
 					<template v-if="!wasPublished && canAddCommunity">
 						<app-form-post-community-pill-add
-							class="-community-pill anim-fade-in-enlarge no-animate-leave"
 							key="add"
+							v-app-scroll-when="scrollingKey"
```

Comment:
well, if you're already at it, move this attribute above the `key` attribute.

Reply:
that's not the styleguide. key is always before since it's very important vue-wise. all of these little changes get auto-fixed with one command, btw.